### ISSUE TITLE
Documentations for AlertManager should be versioned on web.

### ISSUE THREAD

**Issue by kirek007** [prometheus/prometheus#4216]
**Use case. Why is this important?**
Documentation for AlertManager is based on master branch not on last stable release. It's confusing. There're a lot for functionalities that are in docs but can not be used with latest release and there isn't warning about which version this documentation belongs.
## Bug Report
**What did you do?**
Url: https://prometheus.io/docs/alerting/configuration/
**What did you expect to see?**
Docs for latest release
**What did you see instead? Under which circumstances?**
Docs for master branch,

**Comment by juliusv** [comment]
Hi @kirek007, thanks for report! Indeed, that's badly needed. We have an open issue to extract the Alertmanager documentation from the docs repository and move it into the Alertmanager repository, so it can be properly versioned (like the docs for the Prometheus server): https://github.com/prometheus/docs/issues/857

The implementation started at https://github.com/prometheus/alertmanager/pull/1062, but was blocked on some technical dependencies. With a new dedicated docs person starting out in a week, I'm hopeful that this gets fixed soon.

Closing this issue since it's a duplicate of https://github.com/prometheus/docs/issues/857 (or at least will be solved by it).

**Comment by lucperkins** [comment]
@juliusv FYI I have this on my radar 👍

**Comment by lucperkins** [comment]
@juliusv @kirek007 This is currently being addressed [here](https://github.com/prometheus/docs/pull/1084) and [here](https://github.com/prometheus/alertmanager/pull/1441).
